javascript/core/functions-i/function-arguments.md

```diff
@@ -32,27 +26,28 @@ aspects:
 ---
 ## Content
 
-When we pass primitive variables (such as numbers and strings) to functions in JavaScript, they are *passed by value* which means that if we manipulate the argument value inside the function, changes will not be reflected outside the function.
-
-However, if we were to pass an object then changes *are* reflected on the object itself.
+When we pass primitive variables (such as numbers and strings) to functions in JavaScript, they are *passed by value*. This means that if we manipulate the argument value inside the function, changes will not be reflected outside the function.
```

Comment:
I feel like saying "pass by value" immediately might be a bit scary to a beginner (it's an unknown term plus it's arguably kind of a misnomer). 

How about we move it to the end?

```suggestion
When we pass primitive values (such as numbers and strings) into a function, we are actually passing in a copy of each of those values instead of the originals. If we were to manipulate one of those values, the changes will not be reflected on the original value outside of the function. This behavior is commonly known as **pass by value**.
```